go/vt/vtgate/buffer/flags.go

```diff
@@ -70,6 +70,9 @@ func verifyFlags() error {
 	if bufferSize < 1 {
 		return fmt.Errorf("--buffer_size must be >= 1 (specified value: %d)", bufferSize)
 	}
+	if bufferMinTimeBetweenFailovers < 1*time.Second {
+		return fmt.Errorf("--buffer_min_time_between_failovers must be >= 1s (specified value: %v)", bufferMinTimeBetweenFailovers)
+	}
```

Comment:
Should we fail vtgate startup like this? Or just run with the default?
In other words, warn that the 0 value is being ignored and we'll default to 1 minute.

Reply:
It's how we handled the other buffering misconfigurations in that function. I'm fine with changing that behavior generally, but I think it makes sense to be uniform in our handling of them. 

I tend to prefer exiting with an error so that the user is not mistakenly thinking that one configuration is in place (what you told the process to use) while in reality another one is (people don't typically pore over the logs every time they start a process).

Reply:
ok, this is fine.